### PR TITLE
Making memsub promo code lambdas Node 12 compatible.

### PR DESCRIPTION
**Pull request by alexflav23**

### CODE REVIEW

File: lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Data-Lake.js [hunk 1]
```diff
@@ -1,10 +1,7 @@
 'use strict';
 
 const AWS = require('aws-sdk');
-const DOC = require('dynamodb-doc');
-
-const ddb = new AWS.DynamoDB();
-const docClient = new DOC.DynamoDB(ddb);
+const docClient = new AWS.DynamoDB.DocumentClient();
```
Comment (paulbrown1982) [inline lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Data-Lake.js]: Great find!

File: lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js [hunk 2]
```diff
@@ -247,7 +245,7 @@ function makeAPICalls(csvData) {
 exports.handler = (event, context, callback) => {
 
     const TOUCHPOINT_BACKEND = /PROD$/.test(context.functionName) ? 'PROD' : /UAT$/.test(context.functionName) ? 'UAT' : 'DEV';
-    const TableName = `MembershipSub-PromoCode-View-${source}`;
```
Comment (paulbrown1982) [inline lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js]: How did that ever work?!

Comment (alexflav23) [reply lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js]: @paulbrown1982 I don't know to be honest. Maybe older node didn't check properly?

Comment (paulbrown1982) [reply lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js]: The PROD code had drifted from the codebase! Either way, your correction is fine.

```
paul_brown ~/code/memsub-promotions (master) $ git diff
diff --git a/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js b/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
index 2521050..9f2edff 100644
--- a/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
+++ b/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
@@ -246,7 +246,7 @@ function makeAPICalls(csvData) {
 
 exports.handler = (event, context, callback) => {
 
-    const TOUCHPOINT_BACKEND = /PROD$/.test(context.functionName) ? 'PROD' : /UAT$/.test(context.functionName) ? 'UAT' : 'DEV';
+    const source = /PROD$/.test(context.functionName) ? 'PROD' : 'UAT';
     const TableName = `MembershipSub-PromoCode-View-${source}`;
 
     docClient.scan({ TableName })
```

**Pull request merged**